voluseg/_steps/step3.py

```diff
@@ -151,9 +151,9 @@ def add_volume(tuple_name_volume):
             volume_accum.add(volume)
 
         if p.parallel_volume:
-            evenly_parallelize(p.volume_names[timepoints]).foreach(add_volume)
+            evenly_parallelize(np.array(p.volume_names)[timepoints]).foreach(add_volume)
```

Comment:
For consistency, this conversion should be done in `load_parameters`, rather than here.